### PR TITLE
Fingerprint support added

### PR DESCRIPTION
**Pull request by welsinga**
- Also updated to a newer Android SDK version.
- Still need to be corrected/filled for all languages:
- [ ] main_text_instructions
- [ ] upgrade_to_touch_id_title
- [ ] upgrade_to_touch_id_message
- [ ] upgrade_button
- [ ] fingerprint_intro
- [ ] touch_sensor
- [ ] error_auth_wrong_fingerprint
- [ ] error_fingerprint_auth_x_attempts_left
- [ ] error_fingerprint_auth_one_attempt_left

### CODE REVIEW

File: android/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -138,4 +138,12 @@ nur einmalig gültigen Daten einzugeben:</string>
     <Data><![CDATA[<html><body>Gehen Sie zu dem Dienst bei dem Sie sich anmelden wollen und folgen Sie diesen Schritten:<ul> <li>Scannen Sie den QR Code auf der Anmeldeseite des Dienstes.</li> <li><i>Wenn Sie mehrere Identitäten haben, wählen Sie die gewünschte aus.</i></li> <li>Bestätigen Sie Ihre Identität (dazu müssen Sie Ihre PIN eingeben).</li> <li>Sie sind angemeldet!</li> </ul> Drücken Sie auf <b>Scannen</b> um den Anmeldevorgang zu starten.</body></html> ]]> </Data>
   </string>
   <string name="main_text_blocked"><b>Und nun?</b><br></br> Fragen Sie Ihre(n) Identitätsprovider nach einem neuen Aktivierungscode. Zur Kontaktaufnahme mit Ihrem Identitätsprovider(n) überprüfen Sie bitte ihre Identäten (erreichbar von oben rechts). <br></br><br></br> Zusammen mit dem neuen Aktivierungscode erhalten Sie auch Anweisungen mit denen Sie Ihr Benutzerkonto wieder reaktivieren können. Drücken Sie <b>Scannen</b> wenn Sie dazu aufgefordert werden.</string>
+  <string name="upgrade_to_touch_id_title"/>
```
Comment (andrei-egeniq) [inline android/src/main/res/values-de/strings.xml]: Just be sure, adding the string with no value to a language file: does it do nothing or it clears default string? `<string name="upgrade_to_touch_id_title"/>` vs `<string name="upgrade_to_touch_id_title">null</string>` should be the same, and make  `upgrade_to_touch_id_title` null.

Comment (welsinga) [inline android/src/main/res/values-de/strings.xml]: It will do nothing (like a blank value of "").

File: android/build.gradle [hunk 2]
```diff
@@ -29,6 +29,8 @@ dependencies {
     compile 'com.squareup.dagger:dagger:1.2.+'
     provided 'com.squareup.dagger:dagger-compiler:1.2.+'
 
+    //Support
+    compile 'com.android.support:appcompat-v7:23.0.0'
```
Comment (andrei-egeniq) [inline android/build.gradle]: 23.1.1 exists. Any reason for 23.0.0?

**Comment by welsinga** [comment]
No, just a moment in time.

**Comment by andrei-egeniq** [comment]
Look ok, can't merge.

**Pull request merged**